frontend/packages/e2e/tests/e2e/navigation.test.ts

```diff
@@ -80,12 +62,66 @@ test.describe('Navigation and URL Parameters', () => {
       await expectUserTableColumnInAccountsTableVisibility(page, 'hidden')
     })
 
-    test.skip('should handle back/forward navigation with table selection and hiding', async () => {})
-  })
+    test('should handle back/forward navigation with table selection', async ({
+      page,
+    }) => {
+      // Initial state - select accounts table
+      const accountsTable = page.getByRole('button', {
+        name: 'accounts table',
+        exact: true,
+      })
+      await accountsTable.click()
+      await expect(page).toHaveURL(/.*active=accounts/)
+      await expect(accountsTable).toBeVisible()
+
+      // Select users table
+      const usersTable = page.getByRole('button', {
+        name: 'users table',
+        exact: true,
+      })
+      await usersTable.click()
+      await expect(page).toHaveURL(/.*active=users/)
+
+      // Go back to accounts table selection
+      await page.goBack()
+      await expect(page).toHaveURL(/.*active=accounts/)
 
-  test.describe('Parameter Combinations', () => {
-    test.skip('should handle showMode change while table is selected', async () => {})
+      // Go forward to users table selection
+      await page.goForward()
+      await expect(page).toHaveURL(/.*active=users/)
+    })
 
-    test.skip('should handle hiding selected table', async () => {})
+    // FIXME: Browser back on hidden table is not working properly
+    test.skip('should handle back/forward navigation with table hiding', async ({
```

Comment:
This gave me an error, I can't seem to get the accounts table to show up in goBack.
It may have been degrade somewhere.

I would like to create a separate issue to resolve this as a good first issue.